department-of-veterans-affairs/caseflow
# Frontend Code Structure

Connects #5818 

### Description
This PR adds the beginnings of the frontend code.

### Testing Plan
1. Go to /hearings/schedule/build
1. Confirm page loads correctly (just the first few lines of text for now)
